src/shared/logs-show.c

```diff
@@ -485,6 +488,32 @@ static int output_short(
                 fprintf(f, ": [%s blob data]\n", format_bytes(bytes, sizeof(bytes), message_len));
         } else {
                 fputs(": ", f);
+
+                /* URLify config_file string in message, if the message starts with it.
+                 * Skip URLification if the highlighted pattern overlaps. */
+                if (message_len >= config_file_len
+                    && memcmp(message, config_file, config_file_len) == 0 &&
```

Comment:
Not a matter, but I prefer that `&&` is located at the end of the line.

Reply:
Fixed.